bin/install.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+# Official installation script for the secureCodeBox
+# Creates namespace securecodebox-system and installs the operator
+# Then interactively installs all possible resources (scanners, demo-apps, hooks)
+# https://docs.securecodebox.io/
+
+set -eu
+shopt -s extglob
+
+COLOR_HIGHLIGHT="\e[35m"
+COLOR_OK="\e[32m"
+COLOR_ERROR="\e[31m"
+COLOR_RESET="\e[0m"
+
+# @see: http://wiki.bash-hackers.org/syntax/shellvars
+[ -z "${SCRIPT_DIRECTORY:-}" ] \
+  && SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )" \
+  && export SCRIPT_DIRECTORY
```

Comment:
```suggestion
  && SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```
Learned this weekend the hard way that exporting this is usually a bad idea: It overrides the same variable which may be set in an included script and then this subsequent script may fail.